src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -1,5 +1,5 @@
 # We don't properly capitalize SCM in the class name since CreateJob is doing `CLASS_NAME.to_s.camelize.safe_constantize`
```

Comment:
Is this comment still valid, then?

Reply:
At the end of this refactoring, it shouldn't be. I will make sure to drop the comment. :+1: